core/lib/include/bitarithm.h

```diff
@@ -165,6 +165,29 @@ static inline unsigned bitarithm_msb(unsigned v)
 #endif
 }
 
+/**
+ * @brief   Returns the number of leading 0-bits in @p x, starting at the most
+ *          significant bit position.
+ *          If x is 0, the result is undefined.
+ *
+ * @param[in]   x   Input value
+ * @return          Number of leading zero bits
+ */
+static inline uint8_t bitarithm_clzb(uint8_t x)
+{
+#if defined(BITARITHM_HAS_CLZ)
+    /* clz operates on words, so `b` will be promoted to the word size */
```

Comment:
```suggestion
    /* clz operates on unsigned int, so `b` will be promoted to the size of an unsigned int */
```

On 8 bit systems an `unsigned int` still is 16 bit wide.